### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by basil**
A series of code cleanup changes for this repository. Most of these changes fix IntelliJ IDEA warnings. I kept each commit separate. Each commit message should be self-explanatory, but if anything is unclear I am happy to provide further explanation. Also, feel free to revert any specific cleanup commits that are unwanted.

### CODE REVIEW

**Review by res0nance — APPROVED** [review]

File: test/src/test/java/hudson/cli/CancelQuietDownCommandTest.java [hunk 1]
```diff
@@ -92,7 +92,7 @@ public void cancelQuietDownShouldSuccessOnQuietDownedJenkins() throws Exception
     @Test
     public void cancelQuietDownShouldResetQuietReason() throws Exception {
         final String testReason = "reason";
-        j.jenkins.getActiveInstance().doQuietDown(false, 0, testReason);
+        Jenkins.getActiveInstance().doQuietDown(false, 0, testReason);
```
[the remark this answers is not in the document]
Comment (basil) [reply test/src/test/java/hudson/cli/CancelQuietDownCommandTest.java:95]: This is a pre-existing issue that I am explicitly not tackling in this PR.

File: test/src/test/java/hudson/slaves/SlaveComputerTest.java [hunk 2]
```diff
@@ -52,8 +52,8 @@
     @Test
     public void testGetAbsoluteRemotePath() throws Exception {
         //default auth
-        Node nodeA = j.createOnlineSlave();
-        String path = ((DumbSlave) nodeA).getComputer().getAbsoluteRemotePath();
+        DumbSlave nodeA = j.createOnlineSlave();
```
[the remark this answers is not in the document]
Comment (basil) [reply test/src/test/java/hudson/slaves/SlaveComputerTest.java:55]: This is a pre-existing issue that I am explicitly not tackling in this PR.

File: test/src/test/java/hudson/model/NodeTest.java [hunk 3]
```diff
@@ -233,9 +232,9 @@ public void testCanTake() throws Exception {
 
     @Test
     public void testCreatePath() throws Exception {
-        Node node = j.createOnlineSlave();
+        Slave node = j.createOnlineSlave();
```
[the remark this answers is not in the document]
Comment (basil) [reply test/src/test/java/hudson/model/NodeTest.java:235]: This is a pre-existing issue that I am explicitly not tackling in this PR.

File: test/src/test/java/hudson/model/NodeTest.java [hunk 4]
```diff
@@ -194,7 +193,7 @@ public void testGetAssignedLabels() throws Exception {
 
     @Test
     public void testCanTake() throws Exception {
-        Node node = j.createOnlineSlave();
+        Slave node = j.createOnlineSlave();
```
[the remark this answers is not in the document]
Comment (basil) [reply test/src/test/java/hudson/model/NodeTest.java:196]: This is a pre-existing issue that I am explicitly not tackling in this PR.

**Review by StefanSpieker — APPROVED** [review]

**Review by timja — APPROVED** [review]

File: test/src/test/java/hudson/model/NodeTest.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (oleg-nenashev) [reply test/src/test/java/hudson/model/NodeTest.java:196]: Agreed, it does not make the code worse

**Review by oleg-nenashev — APPROVED** [review]
LGTM, terminology is not a blocker, we had classcasts before it

**Comment by MarkEWaite** [comment]
Conversations have concluded.  This PR is now ready for merge.  We will merge it after approximately 24 hours if there is no negative feedback.

**Comment by oleg-nenashev** [comment]
Merging without squash, commits are atomic enough

**Pull request merged**